addons/lunch/views/lunch_product_views.xml

```diff
@@ -193,8 +214,8 @@
                     </div>
                     <group>
                         <group>
-                            <field name="topping_label_1"/>
-                            <field name="topping_quantity_1"/>
+                            <field name="topping_label_1" string='Topping 1 Label'/>
```

Comment:
It is better to do this directly in the python file, this allow to be coherent everywhere, even in the case someone adds this field with studio.